plugin/flags_sources/cmake_file.py

```diff
@@ -75,7 +76,14 @@ def get_flags(self, file_path=None, search_scope=None):
         log.debug("[cmake]:[get]: for file %s", file_path)
         cached_cmake_path = self._get_cached_from(file_path)
         log.debug("[cmake]:[cached]: '%s'", cached_cmake_path)
-        current_cmake_path = self._find_current_in(search_scope, 'project')
+        current_cmake_file = File.search(
+            file_name=self._FILE_NAME,
+            search_scope=search_scope,
+            search_content='project(')
```

Comment:
Did you forget to remove the opening parenthesis here. It breaks the functionality if the CMake project is declared like `project (whatever)`.

Reply:
Oh. You're right. But I did not "miss" it. It was done by design so... In case the word "project" is found in a `CMakeLists.txt` file but is not actually defining a project, say:
```cmake
add_executable(projection_node file.cpp)
```

Let's open an issue for that. That is a bug.